R/slide.R

```diff
@@ -394,14 +379,5 @@ epi_slide <- function(x, f, ..., before, after, ref_time_values,
     x <- unnest(x, !!new_col, names_sep = names_sep)
   }
 
-  # Remove any remaining phony observations. When `all_rows` is TRUE, phony
-  # observations aren't necessarily removed in `slide_one_grp`.
-  if (all_rows) {
-    x <- x[x$.real, ]
-  }
-
-  # Drop helper column `.real`.
-  x$.real <- NULL
-
```

Comment:
Requests regarding other files:
- Update `test-epi_slide.R` "Also checks that we correctly remove extra rows and columns (`.real`) used [...]" comment.
- Maybe review & merge the [`epi_slide` tidying PR](https://github.com/cmu-delphi/epiprocess/pull/399) into this one?
- Bump version in DESCRIPTION, add entry to NEWS.md, merge.

Reply:
I thought version bumping only happens on merge to `main`.

Reply:
True... Eventually on release we will smash these down but it does look like they will be awkward in the mean time.  Some form of automation of the version bump etc. also sounds nice.  We could just do whatever for this PR and fix it up after deciding on an appropriate scheme.